src/AppEntry.tsx

```diff
@@ -7,7 +7,7 @@ import logger from 'redux-logger';
 const AppEntry = () => (
   <Provider
     store={init(
-      ...(process.env.NODE_ENV !== 'production' ? [logger] : [])
+      ...((process.env.NODE_ENV !== 'production' ? [logger] : []) as any)
```

Comment:
I had a look. Usually you can just type the empty array and the full array.

However, redux-logger is providing a different type than redux expects.

```
interface Middleware<
  _DispatchExt = {}, // TODO: see if this can be used in type definition somehow (can't be removed, as is used to get final dispatch type)
  S = any,
  D extends Dispatch = Dispatch
> {
  (api: MiddlewareAPI<D, S>): (next: (action: unknown) => unknown) => (action: unknown) => unknown;
}
```

Redux expects any where you see unknown above. I think this is fine - I don't really see this causing huge problems.